src/components/pages/donation_form/AddressForms.vue

```diff
@@ -7,35 +7,44 @@
 			method="post"
 		>
 			<AutofillHandler @autofill="onAutofill">
-				<name
+				<name-fields
```

Comment:
I think we should be using CamelCase for the name all our custom components in the markup. This will make it consistent with the banners and the existing new components.

Reply:
Yeah, I talked to Co about that and she also likes the PascalCase better. Also, because it's easier to search.